snippets/csharp/VS_Snippets_VBCSharp/csProgGuideDocComments/CS/DocComments.cs

```diff
@@ -553,3 +553,43 @@ interface TestInterface
     }
     //</Snippet15>
 }
+
+//-----------------------------------------------------------------------------
+namespace Wrap16
+{
+    //<Snippet16>
+    // compile with: -doc:DocFileName.xml 
+
+    /// <summary>
+    /// You may have some primary information about this class.
+    /// </summary>
+    public class MainClass
+    {
+    }
+
+    ///<inheritdoc/>
+    public class TestClass: MainClass
+    {
+    }
+    //</Snippet16>
+}
+
+//-----------------------------------------------------------------------------
+namespace Wrap17
+{
+    //<Snippet17>
+    // compile with: -doc:DocFileName.xml 
+
+    /// <summary>
+    /// You may have some primary information about this interface.
+    /// </summary>
+    public interface TestInterface
```

Comment:
Super tiny nit, but to conform to our coding standard:

```suggestion
    public interface ITestInterface
```